pkg/network/server.go

```diff
@@ -381,6 +417,8 @@ func (s *Server) handleGetDataCmd(p Peer, inv *payload.Inventory) error {
 				if err := p.WriteMsg(NewMessage(s.Net, CMDConsensus, cp)); err != nil {
 					return err
 				}
+			} else {
+				log.Warn("can't get payload", hash.String())
```

Comment:
Not sure it's worth a warning (info or debug are better) and this also makes consensus payload handling different from the other types which is not good.

Reply:
It was for debug, not sure if we need to log anything at all.